### PR TITLE
Updated guidelines for custom CSS

### PR DESCRIPTION
**Pull request by Blackbaud-DeniseCisneros**
Updated docs to add custom CSS references to `skyuxconfig.json` file instead of `app-extras.module.ts` file.

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://host.nxt.blackbaud.com/stache?_sp.spa=pr-update-custom-css](https://host.nxt.blackbaud.com/stache?_sp.spa=pr-update-custom-css)

File: src/app/learn/basics/index.html [hunk 1]
```diff
@@ -135,9 +135,16 @@ <h3>
 &#125;
   </sky-code-block>
   <p>
-    After the <sky-code>app.scss</sky-code> file is in place, you need to tell {{ stache.jsonData.global.productNameLong }} where to find your custom CSS. From the <sky-code>src/app</sky-code> folder, open the <sky-code>app-extras.module.ts</sky-code> file and insert this line of code at the beginning of the file.
+    After the <sky-code>app.scss</sky-code> file is in place, you need to tell {{ stache.jsonData.global.productNameLong }} where to find your custom CSS. From the <sky-code>src/app</sky-code> folder, open the <sky-code>skyuxconfig.json</sky-code> file. Under the <sky-code>app</sky-code> section, add the <sky-code>styles</sky-code> property to reference your new SCSS file.
```
Comment (blackbaud-johnly) [inline src/app/learn/basics/index.html:138]: Shouldn't `app.scss` be `app-styles.scss` here too?

Comment (Blackbaud-DeniseCisneros) [reply src/app/learn/basics/index.html:138]: Good catch! Yup, it is. I'll make that change

**Comment by blackbaud-ado** [comment]
Preview build at [https://host.nxt.blackbaud.com/stache?_sp.spa=pr-update-custom-css](https://host.nxt.blackbaud.com/stache?_sp.spa=pr-update-custom-css)

**Comment by blackbaud-ado** [comment]
Preview build at [https://host.nxt.blackbaud.com/stache?_sp.spa=pr-update-custom-css](https://host.nxt.blackbaud.com/stache?_sp.spa=pr-update-custom-css)

**Pull request merged**